pkg/didcomm/protocol/exchange/exchange.go

```diff
@@ -86,29 +91,41 @@ func GenerateInviteWithKeyAndEndpoint(invite *Invitation) (string, error) {
 	return encodedExchangeInvitation(invite)
 }
 
+// Protocol for exchange protocol
+type Protocol struct {
+	transport.OutboundTransport
```

Comment:
This should be a named field. The Protocol struct should not be exposing everything that is in its members.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/a26c159c381cbbb380ef97f616c8f9b9bddd476a..2002cc47490c6dea1ede21a12a1f5ebee834f700#diff-4d3ca323f9b237cfa9940c12efa9b615R96).